sermant-plugins/sermant-tag-transmission/tag-transmission-dubbo2.6.x-plugin/src/main/java/io/sermant/tag/transmission/alibabadubbo/interceptors/AlibabaDubboConsumerInterceptor.java

```diff
@@ -71,6 +72,13 @@ protected void injectTrafficTag2Carrier(RpcInvocation invocation) {
             if (!TagKeyMatcher.isMatch(key)) {
                 continue;
             }
+
+            // if original attachment contains the specific key, then ignore
+            String originalAttachment = invocation.getAttachment(key);
+            if (!StringUtils.isEmpty(originalAttachment)) {
```

Comment:
Check whether the value is null. The same as below.

Reply:
done